openedx/core/djangoapps/kafka_consumer/management/commands/consume_events.py

```diff
@@ -0,0 +1,94 @@
+"""
+Management command for listening to license-manager events and logging them
+"""
+
+import logging
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from confluent_kafka import KafkaError, KafkaException, DeserializingConsumer
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.serialization import StringDeserializer
+from openedx.core.djangoapps.kafka_consumer.consumers import getHandler
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Listen for license events from the event bus and log them. Only run on servers where KAFKA_ENABLED is true
```

Comment:
Isn't this more general, based on command-line args?
```suggestion
    Listen for events from the event bus and log them. Only run on servers where KAFKA_ENABLED is true
```